lib/ui/workoutPlayer/models/Exercise.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:html';
+
+class Exercise {
+  final int id;
+  final String name;
+  final String description;
+  final String thumbnailSrc;
+  final String videoSrc;
+  final int length; // units?
```

Comment:
Let's use Duration instead of int
```suggestion
  final Duration length;
```

Reply:
Noted in Lance's PR but this in seconds!

Reply:
Just checked @ltan02 's PR is also using `final int length;`, and I'd prefer to keep it as an `int` for now to ease the integration process.  

Opened ticket to address this issue in later milestones (https://ubclaunchpad.atlassian.net/browse/WP-84?atlOrigin=eyJpIjoiZjZkMzdhYzNhYTM4NDE5NmIxMzE3ZTg4ZmNiZmIwNzIiLCJwIjoiaiJ9) .